### PR TITLE
ExUI mo 3.0.10

### PR DESCRIPTION
**Pull request by rayliangatos**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/projects/EUI/versions/36928

### Change description ###
https://tools.hmcts.net/jira/projects/EUI/versions/36928

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by Mo-Lala84 — APPROVED** [review]
lgtm

File: api/healthCheck/index.ts [hunk 1]
```diff
@@ -38,41 +39,41 @@ const healthCheckEndpointDictionary = {
...
+        res.send(response);
     } catch (error) {
-        logger.info('error', { healthState: false })
-        res.status(error.status).send({ healthState: false })
+        logger.info('error', { healthState: false });
+        res.status(exists(error, 'status') ? error.status : 500).send({ healthState: false });
     }
 }
-export default router
+export default router;
```
Comment (adnanakgun) [inline api/healthCheck/index.ts:79]: Do the `;` semicolons in this file not break the linting?

File: charts/xui-mo-webapp/values.aat.template.yaml [hunk 2]
```diff
@@ -16,8 +16,8 @@ nodejs:
     #S2S_SERVICE: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
     APPINSIGHTS_INSTRUMENTATIONKEY: ${APPINSIGHTS_INSTRUMENTATIONKEY}
     #FEE_AND_PAY_API: https://payment-api-aat.service.core-compute-aat.internal
-    MANAGE_CASE_LINK: https://xui-webapp-aat.service.core-compute-aat.internal/cases
-    MANAGE_ORG_LINK: https://xui-mo-webapp-aat.service.core-compute-aat.internal/
-    
+    MANAGE_CASE_LINK: https://manage-case.aat.platform.hmcts.net/cases
```
Comment (adnanakgun) [inline charts/xui-mo-webapp/values.aat.template.yaml:19]: Is this desired? Replacing `aat` values with `production` values that has `aat` in its file name?

File: charts/xui-mo-webapp/values.preview.template.yaml [hunk 3]
```diff
@@ -14,8 +14,8 @@ nodejs:
     #SERVICES_TERMS_AND_CONDITIONS_API_SERVICE: http://xui-terms-and-conditions-aat.service.core-compute-aat.internal
     #S2S_SERVICE: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
     #FEE_AND_PAY_API: https://payment-api-aat.service.core-compute-aat.internal
-    MANAGE_CASE_LINK: https://xui-webapp-aat.service.core-compute-aat.internal/cases
-    MANAGE_ORG_LINK: https://xui-mo-webapp-aat.service.core-compute-aat.internal/
+    MANAGE_CASE_LINK: https://manage-case.aat.platform.hmcts.net/cases
```
Comment (adnanakgun) [inline charts/xui-mo-webapp/values.preview.template.yaml:17]: Same as above, but with `preview`/`aat`

**Review by adnanakgun — APPROVED** [review]

**Pull request merged**